### DISCUSSION TITLE
Site Replication return 504

### DISCUSSION THREAD

**Discussion by zhangtao19940520** [minio/minio#17510] [Q&A] [answered]
I deploy minio with docker swarm,but site replicatine failed,error info: 
![image](https://github.com/minio/minio/assets/35730733/8d71a3b3-8861-4ae8-a75d-9b15ba9aeaf2)

can anyone help me resolve this problem?thanks a lot!

**Comment by harshavardhana** [comment, marked as answer]
The user you are using for site replication does not have enough permissions.

**Reply by zhangtao19940520** [reply]
Very cool! Thanks a lot!
